### ISSUE TITLE
Cannot read properties of undefined (reading 'quips')

### ISSUE THREAD

**Issue by romimalta** [tloncorp/landscape-apps#2614]
```
TypeError: Cannot read properties of undefined (reading 'quips')
    at h7 (https://natnex-ronret.tlon.network/apps/groups/assets/index.6996dc80.js:48:226844)
    at Kl (https://natnex-ronret.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:19:52884)
    at mf (https://natnex-ronret.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:8723)
    at pf (https://natnex-ronret.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:947)
    at yg (https://natnex-ronret.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:875)
    at rn (https://natnex-ronret.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:730)
    at sl (https://natnex-ronret.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:21:10884)
    at https://natnex-ronret.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:19:38716
    at e.unstable_runWithPriority (https://natnex-ronret.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:8:3811)
    at _r (https://natnex-ronret.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:19:38494)
```

**Comment by jamesacklin** [comment]
Fixed by #2608

@romimalta Are you on the Tlon internal prerelease source for %garden, %groups, and %talk? I sent an email May 16th on the new sync sources (mentioning the date so you can search easily). This bug should be fixed in the release on those ships.

**Comment by romimalta** [comment]
Yeah I remember that email and ~marnec-dozzod-marnus is my current source.
Going to rejoin in the group and see what happens.

Ty in advance

R

On Jun 15, 2023, at 6:13 PM, james acklin ***@***.******@***.***>> wrote:



Fixed by #2608<https://github.com/tloncorp/landscape-apps/pull/2608>

@romimalta<https://github.com/romimalta> Are you on the Tlon internal prerelease source for %garden, %groups, and %talk? I sent an email May 16th on the new sync sources (mentioning the date so you can search easily). This bug should be fixed in the release on those ships.
